react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -99,6 +99,10 @@
   color: @sk-green-light;
 }
 
+.icon.appliedJobs {
+  color: @sk-green;
```

Comment:
When you have applied for a job and it appears in the search results page, the green used on the applied tick is `@sk-green-light`, feel like we should align these. Thought I should bring that to your attention.